src/Core/Authorization/AuthorizationResolver.cs

```diff
@@ -177,6 +177,12 @@ public bool AreColumnsAllowedForOperation(string entityIdentifier, string roleNa
             return false;
         }
 
+        /// <inheritdoc />
+        public bool IsDBPolicyDefinedForRoleAndAction(string entityName, string roleName, EntityActionOperation operation)
+        {
+            return !string.IsNullOrEmpty(GetDBPolicyForRequest(entityName, roleName, operation));
```

Comment:
We should rather use `string.isNullOrWhiteSpace` here. Since now this kind of determines 1 Vs 2 db round trips.